src/components/Home.js

```diff
@@ -0,0 +1,31 @@
+import PageLinks from './PageLinks';
+import Header from './Header';
+import '../styles/HomePage.css';
+
+const Home = () => (
```

Comment:
- [ ] It would be better if we implement the filter functionality where we can filter by company names in our case as we have in the requirements
![Screenshot from 2023-06-27 23-05-05](https://github.com/ClintonYounge/Mobile-Tech-Financial-Digest/assets/46388113/4b8a5b2b-2315-4a36-ad0a-9638601eb239)